src/Command.js

```diff
@@ -19,6 +19,47 @@ export default class Command {
     this.concurrency = (!flags || flags.concurrency === undefined) ? DEFAULT_CONCURRENCY : Math.max(1, +flags.concurrency || DEFAULT_CONCURRENCY);
   }
 
+  get name() {
+    // For a class named "FooCommand" this returns "foo".
+    return this.className.replace("Command", "").toLowerCase();
+  }
+
+  get className() {
+    return this.constructor.name;
```

Comment:
You would prefer each command implement a name getter?

Like:

```javascript
class LsCommand extends Command {
  ...
  get name() { return "ls" }
}
```

That's more explicit and flexible... but it's a lot of boilerplate, too.